.github/workflows/openqa.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           echo build=$(openqa-cli api
           --host ${OPENQA_HOST:-https://openqa.opensuse.org}
           job_groups/${OPENQA_BUILD_LOOKUP_GROUP_ID:-1}/build_results only_tagged=1
-          | jq -r '[ .build_results[] | select(.tag.description=="published") | .build ][0]'
+          | jq -e -r '[ .build_results[] | select(.tag.description=="published") | .build ][0]'
```

Comment:
Small niptick: Consider usage of `jq` always as a code smell, that's why you have apis for :) (I think I've said it before)

Reply:
Yes you have but I don't see how that should be related to this PR.